lib/css/modules/_showImages.scss

```diff
@@ -517,6 +517,38 @@ body:not(.res-showImages-displayImageCaptions) {
 	}
 }
 
+.res-iframe-expando {
+
+	&-drag-handle {
```

Comment:
If you wrap the drag handle and `siteAttribution` (`<div class="res-expando-siteAttribution"></div>`, which `addSiteAttribution` will detect) in a new element with rules
```
    display: flex;
    flex-direction: row-reverse;
    justify-content: space-between;
```